component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/constants/BackupCodeAuthenticatorConstants.java

```diff
@@ -48,6 +48,12 @@ public class BackupCodeAuthenticatorConstants {
     public static final String IS_INITIAL_FEDERATED_USER_ATTEMPT = "isInitialFederationAttempt";
     public static final String CODE_MISMATCH = "codeMismatch";
     public static final String GET_PROPERTY_FROM_IDENTITY_CONFIG = "getPropertiesFromLocal";
+    public static final String MAX_ATTEMPTS_EXCEEDED = "MAX_ATTEMPTS_EXCEEDED";
+    public static final String ADMIN_INITIATED = "ADMIN_INITIATED";
+    public static final String CONF_SHOW_AUTH_FAILURE_REASON = "showAuthFailureReason";
+    public static final String CONF_SHOW_AUTH_FAILURE_REASON_ON_LOGIN_PAGE = "showAuthFailureReasonOnLoginPage";
+    public static final String ERROR_CODE = "errorCode";
+    public static final String LOCKED_REASON = "lockedReason";
```

Comment:
check if there are constants already defined for these in the framework repo

Reply:
Possible framework constants are used in 9fc302cacfce6ab5b301cf7789da04bd5f7816cf